### PR TITLE
DropDownEditors: Cover components with tests with testAccessibility

### PR DESCRIPTION
**Pull request by marker-dao**

### CODE REVIEW

File: packages/devextreme/testing/testcafe/tests/accessibility/autocomplete.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+import { defaultSelector, testAccessibility, Configuration } from '../../helpers/accessibility/test';
+import { Options } from '../../helpers/generateOptionMatrix';
+import { Properties } from '../../../../js/ui/autocomplete.d';
+import Autocomplete from '../../model/autocomplete';
+
+fixture.disablePageReloads`Accessibility`
+  .page(url(__dirname, '../container.html'))
+  .afterEach(async () => clearTestPage());
+
+const items = ['A', 'Aa', 'B'];
```
Comment (EugeniyKiyashko) [inline packages/devextreme/testing/testcafe/tests/accessibility/autocomplete.ts]: It's better to choose more informative item's name as follows
const items = ['Item_1', 'Item_2', 'Item_3'];

Comment (marker-dao) [reply packages/devextreme/testing/testcafe/tests/accessibility/autocomplete.ts]: [1d96c8aa2cf8bac45468c91f917d9e162d46d2bc]

**Review by alexanderPolosatov — DISMISSED** [review]

File: .vscode/settings.json [hunk 2]
```diff
@@ -3,6 +3,6 @@
          "watch": true,
     },
     "editor.codeActionsOnSave": {
-        "source.fixAll.eslint": true
+        "source.fixAll.eslint": "explicit"
```
Comment (ksercs) [inline .vscode/settings.json]: Is it intended? Why we do this?

File: .github/workflows/testcafe_tests.yml [hunk 3]
```diff
@@ -62,9 +62,9 @@ jobs:
       fail-fast: false
       matrix:
         ARGS: [
-          { componentFolder: "accessibility", name: "accessibility" },
-          { componentFolder: "accessibility", name: "accessibility - material", theme: 'material.blue.light' },
-          { componentFolder: "accessibility", name: "accessibility - fluent", theme: 'fluent.blue.light' },
+          { componentFolder: "accessibility", name: "accessibility", concurrency: 2 },
```
Comment (ksercs) [inline .github/workflows/testcafe_tests.yml]: Why?

File: .vscode/settings.json (hunk 2 above)
Comment (marker-dao) [reply .vscode/settings.json]: My version of VS Code changed these settings on its own. I will now revert back to the previous version.

Comment (marker-dao) [reply .vscode/settings.json]: [1dcfb467fcbcf0d07992d1c4023b3c01dbed1eb0]

File: .github/workflows/testcafe_tests.yml (hunk 3 above)
Comment (marker-dao) [reply .github/workflows/testcafe_tests.yml]: We tried to check if the test execution time would decrease if we set the value to 2. We can remove this value now, as there is no noticeable effect.

Comment (marker-dao) [reply .github/workflows/testcafe_tests.yml]: [f3e0583b6c68591e7ea000377b1d886c0e1f85ee]

**Review by ksercs — APPROVED** [review]

**Review by EugeniyKiyashko — APPROVED** [review]

**Pull request merged**